bin/interpreter.sh

```diff
@@ -112,34 +113,20 @@ if [[ "${INTERPRETER_ID}" == "spark" ]]; then
       addJarInDirForIntp "${HADOOP_HOME}/lib"
     fi
 
-    addJarInDirForIntp "${INTERPRETER_DIR}/dep"
+    # If there is not SPARK_HOME in the system, Zeppelin will use local Spark binary for Spark interpreter
+    export SPARK_SUBMIT="${ZEPPELIN_HOME}/${SPARK_CACHE}/${SPARK_ARCHIVE}/bin/spark-submit"
 
-    pattern="${ZEPPELIN_HOME}/interpreter/spark/pyspark/py4j-*-src.zip"
+    pattern="${ZEPPELIN_HOME}/${SPARK_CACHE}/${SPARK_ARCHIVE}/python/lib/py4j-*-src.zip"
     py4j=($pattern)
     # pick the first match py4j zip - there should only be one
-    PYSPARKPATH="${ZEPPELIN_HOME}/interpreter/spark/pyspark/pyspark.zip:${py4j[0]}"
+    PYSPARKPATH="${ZEPPELIN_HOME}/${SPARK_CACHE}/${SPARK_ARCHIVE}/python/lib/pyspark.zip:${py4j[0]}"
 
     if [[ -z "${PYTHONPATH}" ]]; then
       export PYTHONPATH="${PYSPARKPATH}"
     else
       export PYTHONPATH="${PYTHONPATH}:${PYSPARKPATH}"
     fi
     unset PYSPARKPATH
-
-    # autodetect HADOOP_CONF_HOME by heuristic
-    if [[ -n "${HADOOP_HOME}" ]] && [[ -z "${HADOOP_CONF_DIR}" ]]; then
-      if [[ -d "${HADOOP_HOME}/etc/hadoop" ]]; then
-        export HADOOP_CONF_DIR="${HADOOP_HOME}/etc/hadoop"
-      elif [[ -d "/etc/hadoop/conf" ]]; then
-        export HADOOP_CONF_DIR="/etc/hadoop/conf"
-      fi
-    fi
-
-    if [[ -n "${HADOOP_CONF_DIR}" ]] && [[ -d "${HADOOP_CONF_DIR}" ]]; then
-      ZEPPELIN_INTP_CLASSPATH+=":${HADOOP_CONF_DIR}"
-    fi
-
-    export SPARK_CLASSPATH+=":${ZEPPELIN_INTP_CLASSPATH}"
```

Comment:
@Leemoonsoo 
There are two type of code blocks I removed. One is for exporting `HADOOP_CONF_DIR` and the other is `SPARK_CLASSPATH`. As you know, those two are all for the order Spark version support. 
 - Regarding the setting `HADOOP_CONF_DIR` is not the problem for this change since it's embraced in the if statement(maybe we can remove this in another PR as refactoring). Anyway I reverted this part in [f5dcd04e](https://github.com/apache/zeppelin/pull/1339/commits/f5dcd04e1ec1e9575bc8d8a8e858b0b83f60f578).
 
- But `SPARK_CLASSPATH` can conflict with `SPARK_SUBMIT` that I used in [here](https://github.com/apache/zeppelin/pull/1339/files#diff-8d3bd42a5de342ebad38e1d113d9f75cR117). `SPARK_CLASSPATH` is deprecated after `spark-1.x` and  `SPARK_SUBMIT` is recommended instead. 
```
Welcome to
      ____              __
     / __/__  ___ _____/ /__
    _\ \/ _ \/ _ `/ __/  '_/
   /___/ .__/\_,_/_/ /_/\_\   version 1.0.0
      /_/

Using Scala version 2.10.4 (IBM J9 VM, Java 1.7.0)
WARN spark.SparkConf: 
SPARK_CLASSPATH was detected (set to 'path-to-proprietary-hadoop-lib/*:/path-to-proprietary-hadoop-lib/lib/*').
This is deprecated in Spark 1.0+.

Please instead use:
 - ./spark-submit with --driver-class-path to augment the driver classpath
 - spark.executor.extraClassPath to augment the executor classpath
```
Please see [this mail thread](http://apache-spark-user-list.1001560.n3.nabble.com/SPARK-CLASSPATH-Warning-td9233.html) for the more information about this. 
Since I set the local spark version as `spark-2.0.0`, we need to use only one : `SPARK_CLASSPATH` or `SPARK_SUBMIT`. That's why I removed the `SPARK_CLASSPATH` and set `SPARK_SUBMIT`.